cmd/bacalhau/docker_run.go

```diff
@@ -36,6 +36,7 @@ var jobCPU string
 var jobMemory string
 var jobGPU string
 var skipSyntaxChecking bool
+var isLocal bool
```

Comment:
@js-ts Should we clean `isLocal` at https://github.com/filecoin-project/bacalhau/blob/eff3486ece33c0da3d758c65ee7930de0003debc/cmd/bacalhau/docker_run.go#L382 as well?

Reply:
you mean add this there ?
`isLocal = false`

Reply:
added 
isLocal = false